lib/career_bot/responders/processed_message_responder.rb

```diff
@@ -0,0 +1,51 @@
+class ProcessedMessageResponder
```

Comment:
I found something here. You moved `deliver_messages` to this class. In ApplyResponder `deliver_messages` is not working now, since it's moved. I would suggest removing nesting from `BotResponder` class in `ApplyResponder` class and adding two following methods: 

```ruby
  def deliver_messages(message)
    FacebookMessenger.new.deliver(conversation.messenger_id, message)
  end

  def repository
    @repository ||= ConversationRepository.new
  end
```
That will resolve the problem for now. Then, in next phase of refactoring we could add handling sending message in one place instead of initializing Conversation repo & FacebookMessenger class in many places :(